lib/myFTClient.js

```diff
@@ -676,6 +718,17 @@ function getGroupsFollowingConcept(licenceId, conceptId) {
 }
 
 /**
+ * Gets the users that are following a concept/topic as an individual
+ * @param {String} licenceId -
+ * @param {String} conceptId -
+ * @returns {Promise} response -
+ */
+function getUsersFollowingConceptAsIndividual(licenceId, conceptId) {
```

Comment:
We can maybe rename this to `getUsersFollowingConceptByKat` and have an extra param for the follow type because we need to be able to extract users that have the new `followed_by_kat` relationship with `individual_follow=true`, `individual_follow=false` or just ignore `individual_follow`.
So something like:
```
function getUsersFollowingConceptAsIndividual(licenceId, conceptId, followType) {
  let params;
  if (followType) {
    params = { followType };
  }
  return _getNodesFollowingConceptByKat(licenceId, conceptId, myftConst.userNodeName, params);
}
```